specification/src/main/asciidoc/platform/PreviousVersionDeploymentDescriptors.adoc

```diff
@@ -3,21 +3,36 @@
 == Previous Version Deployment Descriptors
 
 This appendix describes Document Type
-Definitions and XML schemas for Deployment Descriptors from previous
-versions of the Java EE specification. All Jakarta EE products are required
-to support these DTDs and schemas as well as the schemas specified in
-this version of the specification. This ensures that applications
-written to previous versions of this specification can be deployed on
-products supporting the current version of this specification. In
-addition, there are no restrictions on mixing versions of deployment
+Definitions (DTDs) and XML schemas for Deployment Descriptors from previous
+versions of the Java(TM) EE and Jakarta(TM) EE specifications.
+All Jakarta EE 9 products are required
+to support the DTDs and schemas specified by Jakarta EE 8,
+as well as the schemas specified in this version of the specification. 
+This ensures that applications
+written to the previous version of the Jakarta EE specification can be deployed on
+products supporting the current version of this specification. 
+Support for versions of DTDs and schemas prior to Jakarta EE 8 is optional.
+In addition, there are no restrictions on mixing versions of supported deployment
 descriptors in a single application; any combination of valid deployment
 descriptor versions must be supported.
 
-=== Java EE 7 Application XML Schema
+=== Jakarta EE 9 schemas
 
```

Comment:
This document describes about **previous** versions of DTDs.
Is it better to move Jakarta EE 9 Schemas to "ApplicationAssemblyDeployment.adoc" ?

Reply:
@kazumura, I realize that this section was for "past versions", but due to the major change in namespace, I thought it wouldn't hurt to get this in front of people regardless of the section.  Having it in two locations sounds okay for Jakarta EE 9.